app/src/gecko/java/org/mozilla/tv/firefox/ext/EngineView.kt

```diff
@@ -29,15 +26,15 @@ fun EngineView.setupForApp() {
     // Also increase text size to fill the viewport (this mirrors the behaviour of Firefox,
     // Chrome does this in the current Chrome Dev, but not Chrome release).
     // TODO #33: TEXT_AUTOSIZING does not exist in AmazonWebSettings
-    // webView.settings.setLayoutAlgorithm(AmazonWebSettings.LayoutAlgorithm.TEXT_AUTOSIZING);
+    // geckoView.settings.setLayoutAlgorithm(AmazonWebSettings.LayoutAlgorithm.TEXT_AUTOSIZING);
 
-    if (BuildConstants.isDevBuild) {
-        WebView.setWebContentsDebuggingEnabled(true)
-    }
+//    if (BuildConstants.isDevBuild) {
+//        GeckoView.setWebContentsDebuggingEnabled(true)
+//    }
 
-    // WebView can be null temporarily after clearData(); however, activity.recreate() would
-    // instantiate a new WebView instance
-    webView?.setOnFocusChangeListener { _, hasFocus ->
+    // GeckoView can be null temporarily after clearData(); however, activity.recreate() would
+    // instantiate a new GeckoView instance
```

Comment:
Is this comment accurate for GeckoView?

Reply:
No, I actually haven't changed comments in gv/EngineView.kt. Will update!